examples/nodemailer/src/index.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import { Email } from './email';
 
 const transporter = nodemailer.createTransport({
-  host: 'smtp.ethereal.email',
+  host: 'smtp.ethereal.email' || process.env.CUSTOMHOST_URL,
```

Comment:
I think the env var should come first? Otherwise if won't replace the actual value when a user pass the env vars.

```suggestion
  host: process.env.CUSTOMHOST_URL || 'smtp.ethereal.email',
```